repository_service_tuf/cli/admin/import_targets.py

```diff
@@ -0,0 +1,168 @@
+                {
+                    "path": line.split(";")[0],
+                    "info": {
+                        "length": int(line.split(";")[1]),
+                        "hashes": {line.split(";")[2]: line.split(";")[3]},
+                    },
+                    "rolename": succinct_roles.get_role_for_target(
+                        line.split(";")[0]
+                    ),
+                    "published": False,
+                    "action": "ADD",
+                    "last_update": datetime.now(),
+                }
+            )
+
+    return rstuf_db_data
+
+
+def _import_csv_to_rstuf(
+    db_client: Connection,
+    rstuf_table: Table,
+    csv_files: List[str],
+    succinct_roles: SuccinctRoles,
+) -> None:
+    for csv_file in csv_files:
+        console.print(f"Import status: Loading data from {csv_file}")
+        rstuf_db_data = _parse_csv_data(csv_file, succinct_roles)
+        console.print(f"Import status: Importing {csv_file} data")
+        try:
+            db_client.execute(rstuf_table.insert(), rstuf_db_data)
+        except IntegrityError:
+            raise click.ClickException(
+                "Import status: ABORTED due duplicated targets. "
+                "CSV files must to have unique targets (path). "
+                "No data added to RSTUF DB."
+            )
+        console.print(f"Import status: {csv_file} imported")
+
+
+@admin.command()
+@click.option(
+    "-metadata-url",
+    required=True,
+    help="RSTUF Metadata URL i.e.: http://127.0.0.1 .",
+)
+@click.option(
+    "-db-uri",
+    required=True,
+    help="RSTUF DB URI. i.e.: postgresql://postgres:secret@127.0.0.1:5433",
+)
+@click.option(
+    "-csv",
+    required=True,
+    multiple=True,
+    help="CSV file to import. Multiple -csv parameters are allowed.",
+)
+@click.option(
+    "--skip-publish-targets",
+    is_flag=True,
+    help="Skip publish targets process in TUF Metadata.",
+)
+@click.pass_context
+def import_targets(context, metadata_url, db_uri, csv, skip_publish_targets):
+    """
+    Import targets to RSTUF from exported CSV file.
+    """
+    settings = context.obj["settings"]
+    server = settings.get("SERVER")
+    token = settings.get("TOKEN")
+    if server and token:
+        token_access_check = is_logged(server, token)
+        if token_access_check.state is False:
+            raise click.ClickException(
+                f"{str(token_access_check.data)}"
+                "\n\nTry re-login: 'Repository Service for TUF admin login'"
+            )
+    else:
+        raise click.ClickException("Login first. Run 'rstuf admin login'")
+
+    headers = get_headers(settings)
+
+    response = request_server(metadata_url, "1.bin.json", Methods.get)
+    if response.status_code == 404:
+        raise click.ClickException("RSTUF Metadata Targets not found.")
+
+    # load all required infrastructure
+    json_data = json.loads(response.text)
+    targets = Metadata.from_dict(json_data)
```

Comment:
You can use insted the `JSONDeserializer()` from `python-tuf`: https://github.com/theupdateframework/python-tuf/blob/2840f629b07409b610c8943a2614272f0ff02bbc/tuf/api/serialization/json.py#L30

Also, because you are relying on a specific python-tuf `Signed` type it will be good to add an annotation for `targets` like:
`targets: Metadata[Targets] = Metadata.from_dict(json_data)`